storages/factory.go

```diff
@@ -89,7 +90,7 @@ func Create(ctx context.Context, name, logEventPath string, destination Destinat
 		return nil, nil, fmt.Errorf("Unknown destination mode: %s. Available mode: [%s, %s]", destination.Mode, BatchMode, StreamMode)
 	}
 
-	processor, err := schema.NewProcessor(tableName, mapping)
+	processor, err := schema.NewProcessor(tableName, mapping, destination.DataLayout.MappingType)
```

Comment:
potential panic. destination.DataLayout can be nil.

Reply:
Thnx for noticing! Fixed